### ISSUE TITLE
Remove domain specific logic from "component library"

### ISSUE THREAD

**Issue by markus-heer** [byteleaf/companyon-frontend#37]
At the moment we mix generic components with Domain specific ones. My idea would be to separate a "component library folder" with atoms, molecules and organisms (or another structure, no strong opinion on this from my side) that is fully generic and without domain logic. Additionally we could have a components folder where we put all domain specific stuff. What do you think about this?

Perhaps rather a big overhead, but perhaps we could also introduce Lerna for this. In the future we will definitely have modules that need to be separate libs anyways, so perhaps we could start with it right away to get used to it?

**Comment by apotdevin** [comment]
Agree, let's move components with business logic out of this folder.

Ideas:
1. Move generic components to a separate `libs` folder on the root level (Although I think CRA complains about this)
2. Create a new components folder inside `src` and move either generic or business comps here

Regarding Lerna I think we should also clarify how do we want to package backend+frontend for the future. If the idea is to have backend provide the frontend then maybe we should also create a unified repo for both. Lerna would also greatly help here.
